.github/workflows/deploy.yml

```diff
@@ -82,22 +82,16 @@ jobs:
           helm repo add bitnami-pre-2022 https://raw.githubusercontent.com/bitnami/charts/eb5f9a9513d987b519f0ecd732e7031241c50328/bitnami
           helm repo add stable https://charts.helm.sh/stable/
           helm dependency build ./helm/api-platform
-      - name: Check for existing namespace
-        id: k8s-namespace
-        run: echo "::set-output name=namespace::$(kubectl get namespace ${{ inputs.namespace }} 2> /dev/null)"
       # Release name MUST start with a letter
-      # GitHub doesn't support multilines environment variables (JWT_*_KEY)
-      - name: Deploy in new namespace
-        if: steps.k8s-namespace.outputs.namespace == ''
+      - name: Deploy
         run: |
-          JWT_PASSPHRASE=$(openssl rand -base64 32)
-          JWT_SECRET_KEY=$(openssl genpkey -pass file:<(echo "$JWT_PASSPHRASE") -aes256 -algorithm rsa -pkeyopt rsa_keygen_bits:4096)
           helm upgrade ${{ inputs.release }} ./helm/api-platform \
             --reuse-values \
```

Comment:
As you no longer need to reuse `JWT_*` variables, I think you can remove the `--reuse-values` flag, it will help to apply other changes from `values.yaml`